R/create_figures_doc.R

```diff
@@ -17,12 +17,31 @@ create_figures_doc <- function(resdir = NULL,
                                year = NULL,
                                subdir = NULL,
                                include_all = TRUE) {
+
   model <- match.arg(model, several.ok = FALSE)
 
+  # import pre-written captions and alt text
   captions_alttext <- utils::read.csv(
     system.file("resources", "captions_alttext.csv", package = "asar")
   )
 
+  # import converted model output
+  output <- utils::read.csv(
+    paste0(resdir, "/", model_results, ".csv")
+  )
+
+  # output <- utils::read.csv(
+  #   "petrale_convert_output.csv"
+  # )
+
+  # extract quantities
+  # using the <<- exports the object to the R environment!
+  start_year <<- as.numeric(output[5,5])
+
+  Fend <<- as.numeric(output[2,2])
```

Comment:
ditto here. Also why are you exporting it to the R environment?

Reply:
Exporting to the R environment makes it easy for the user to see what the parameters are and eliminates the need for them to run the skeleton's output_and_quantities chunk in order to obtain those values

Reply:
Please do not populate the R environment with objects. This is poor practice for R coding and makes it difficult to render multiple assessments in one R session.